utils/terraform_client.py

```diff
@@ -206,7 +198,8 @@ def apply(self):
     def terraform_apply(self, apply_spec):
         name = apply_spec['name']
         tf = apply_spec['tf']
-        return_code, stdout, stderr = tf.apply(auto_approve=True,
+        return_code, stdout, stderr = tf.apply(dir_or_plan=name,
```

Comment:
why do you have a plan and then you do a `skip_plan` afterwards?

Reply:
skip_plan is really just to enable auto-approve:
https://github.com/beelit94/python-terraform/blob/a33aebce1ca34357bb8a999f9853a6f614fba924/python_terraform/__init__.py#L112

Reply:
There's no need for auto-approve if you apply a plan, it doesn't ask. But if you skip the plan you've already calculated, I'm afraid it will calculate a new one...

Reply:
that makes sense. i've removed `auto_approve` and `skip_plan`. we'll see if it behaves as we expect.